### ISSUE TITLE
Fighter balancing turret suggestion 

### ISSUE THREAD

**Issue by KyroFirehammer** [endless-sky/endless-sky#2979]
I am currently playing through the wanderers storyline an am noticing that on both sides the drones/fighters/both-at-the-same-time (looking at you sestor) and many of the "good on fighters" weapons are really lacking because you have to target each ship individually which is highly inneffectve against large swarms of them. What i suggest is a turret (or turrets so more than one species has them) that works like a regular turret but deals weaker damage ,but has massive AOE and targets fighters and drones as a priority (Basically this weapon is garbage on anything bigger than a light warship  but is great against fleets of drones fighters intersepeters and weaker light warships) this would help large leadership from not being super weak to small ships in large numbers.

**Comment by MCOfficer** [comment]
something like a Flak Cannon, then?

the invention of turret turn speed  was also (in parts) a try to buff fighters (reward mobility), at least that's how i understood it. a Flak turret would kinda counter this, unless it deals really low damage.

**Comment by KyroFirehammer** [comment]
The main reason for this would be to have a turret that would defend from fighters while you deal with their mothership, another good way to fix this would be to have a feature where you could assign individual turrets priorities (default would be your be your target) so you could assign turrets to defend your ship from fighters while still fighting the main ship (swarms would still be effective, but less obnoxious to fight against without your own fighters)

**Comment by Amazinite** [comment]
I wouldn't see this being justified unless the effective DPS of turrets was so low that we could make a flak cannon weapon that wouldn't be overpowered thanks to its damage and pretty much guaranteed tracking. E.g. if the Laser Turret's effective DPS against fighters was like 50% of what its actual DPS is, but currently I don't think that turret turn rates are slow enough for a dedicated anti-fighter weapon to be justified.

**Comment by tehhowch** [comment]
The only part of this suggestion that cannot be done currently is the preferential selection of a ship based on its classification.

**Comment by KyroFirehammer** [comment]
I thought so that's why I put the turret I was thinking this thing would be sorta like a AM where it only hits small ships, or it did horrid damage that would prioritize fighting fighters and drones

**Comment by KiLEdEnNis** [comment]
a turret w/ a submunition, a small random lifetime (think that's possible), short range, high damage and a blast radius gives that effect, especially with opportunistic personality/setting

**Comment by MCOfficer** [comment]
either way, this kinda turret would only be useful
a) in a Fleet where 1 ship specialises on Flak, this targeting fighters
b) if turrets that cannot reach the current target automatically track a closer target

apart from that, most times it would be rendered useless by the player aiming for bigger ships.

**Comment by KiLEdEnNis** [comment]
b is the scenario you'd want; fighter swarms get close, so close-range high AoE is a lot of damage

**Comment by KyroFirehammer** [comment]
Thats the idea I was going for

**Comment by tehhowch** [comment]
Ships will not fire weapons which can hurt themselves. Note that human level tech is extremely unlikely to be so sophisticated that it is able to harm only hostiles - e.g. if you use `safe` you are pretty much making a weapon that either is 1) not human in origin, or 2), not going to be used outside of a mod.

**Comment by KyroFirehammer** [comment]
Oh I was thinking along the lines of mereti or wanderer tech

**Comment by MessyMix** [comment]
Go with lower AoE to solve the problem of self-damage. Or, make it Remnant
tech, although no alien race (save the Drak) has `safe` currently, so
there's no saying who they looted their tech from...

**Comment by KyroFirehammer** [comment]
I personally think it would make sense to me mereti because of the Leks and Osks in the sestor fleet

Pug would also make sense on the mabotos and atfectas

**Comment by Elyssaen** [comment]
Replaying EV Nova a while ago, I got thinking about an alternative that would also help satisfy the camp that argue that it's a waste to install an anti-missile turret. 

In EV Nova, anti-missile weapons also independently shot at fighters. What _might_ be interesting would be an extension as follows:

1. An anti-missile turret will automatically prioritise shooting down a missile.
2. If there are no missiles in range, but there are drones/fighters, it will automatically fire on those.
3. If there are none of both, then it might respond to your primary fire trigger and aim at your target like a normal turret.

But, all this said, I don't think it's hugely important. Fighters are pesky but not really any stronger than they should be, so this'd just be a convenience. The flak weapon idea is definitely a fun one to explore for an alien sometime, so long as the tech originated from Tier 2+.

**Comment by KiLEdEnNis** [comment]
the problem is that AM is useless in most cases and basically just cancels out missiles. so something like that would make AM more attractive as they have an actual use in every case; I'm for this change, or at least something along the way of "not useless if there are no missiles"

**Comment by gunqqer** [comment]
I feel like missile and AM need a rework. I don't use either of them. I don't use missiles because of the limited ammo that is hard to get, most have eh tracking, and can be shot down. I don't use AM because I could use that space for other things and my ships can normaly just tank the shots. The only missiles that I would want AM for are the finisher missiles (or is that the ship? idk) just because of the raw damage they output and coalition space doens't have many missiles. 

How are these fixed? I really don't know

**Comment by KyroFirehammer** [comment]
Agreed high end ammo is too hard to get my idea from #2887 would fix the ammo, and if the AM did double as a fighter turret would be a good fix however it would have to take numerous shots to kill (for balancing purposes)
